stable/grafana/templates/secret.yaml

```diff
@@ -15,3 +15,6 @@ data:
   {{- else }}
   admin-password: {{ randAlphaNum 40 | b64enc | quote }}
   {{- end }}
+  {{ if and .Values.ldap.enabled .Values.ldap.createConfig }}
+  ldap.toml: {{ include "ldap.toml" . | b64enc | quote }}
```

Comment:
I know using dots in keys is supported but is also cumbersome, can we use something like `ldap-toml` and then when specifying the key and path to use for the volume use `key: ldap-toml`, `path:ldap.toml`

Reply:
updated